Add share button

Nazeeh21 opened the pull request:
Closes #263

farque65 commented:
Two things to add to this.
- The share button should also be visible on the host view
- A notification should pop up saying the room link has been copied

Nazeeh21 commented:
@farque65 I've added the share button for the Host room and also added the notification when the link gets copied successfully. Pls have a look and let me know if any of the changes are required.

farque65 commented:
Looks good. I just updated the notification to be more consistent with the rest of the notifications on tip.party and button placement
